packages/sources/coinmarketcap/README.md

```diff
@@ -392,3 +392,89 @@ https://pro-api.coinmarketcap.com/v1/cryptocurrency/quotes/latest
   "statusCode": 200
 }
 ```
+
+## Historical Endpoint
+
+https://pro-api.coinmarketcap.com/v1/cryptocurrency/quotes/historical
+
+### Input Params
+
+| Required? |                  Name                   |                                   Description                                   |                                          Options                                          | Defaults to |
+| :-------: | :-------------------------------------: | :-----------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------: | :---------: |
+|    ✅     | `base`, `from`, `coin`, `symbol`, `sym` |                       The symbol of the currency to query                       |                                                                                           |    `BTC`    |
+|           |   `quote`, `to`, `market`, `convert`    |                    The symbol of the currency to convert to                     |                                                                                           |    `USD`    |
```

Comment:
It seems like this input param is not defaulting to "USD", and should be required

Reply:
Same goes for the above, should not be default "BTC"